lib/lrama/reporter/states.rb

```diff
@@ -15,273 +15,337 @@ def initialize(itemsets: false, lookaheads: false, solved: false, counterexample
 
       # @rbs (IO io, Lrama::States states) -> void
       def report(io, states)
-        if @counterexamples
-          cex = Counterexamples.new(states)
-        end
+        cex = Counterexamples.new(states) if @counterexamples
 
         states.states.each do |state|
           # Report State
```

Comment:
This comment is for `io << "State #{state.id}\n\n"` then can remove now.

Reply:
Thank you! remove in: https://github.com/ruby/lrama/pull/582/commits/bfa580159e8e51fe70aa5cf5c50c24f9ba9975c6